extension/test/integration/get-payment-methods.handler.spec.js

```diff
@@ -5,13 +5,14 @@ import c from '../../src/config/constants.js'
 import config from '../../src/config/config.js'
 import utils from '../../src/utils.js'
 
-describe('::getPaymentMethods::', async () => {
-  const packageJson = await utils.readAndParseJsonFile('package.json')
```

Comment:
What's the actual benefit to put the initialization under `before()`?

Reply:
Benefit is that it didn't work. I fixed it and now it works. So this is the fix. See the explanation here in the Description of this PR: https://github.com/commercetools/commercetools-adyen-integration/pull/970